plugins/inputs/lustre2/lustre2.go

```diff
@@ -432,6 +455,90 @@ func (l *Lustre2) GetLustreProcStats(fileglob string, wantedFields []*mapping) e
 	return nil
 }
 
+func (l *Lustre2) GetLustreProcBrwStats(fileglob string, wantedFields []*mapping) error {
+	files, err := filepath.Glob(fileglob)
+	if err != nil {
+		return err
+	}
+
+	for _, file := range files {
+		// Turn /proc/fs/lustre/obdfilter/<ost_name>/stats and similar into just the object store target name
+		// This assumes that the target name is always second to last, which is true in Lustre 2.1->2.12
+		path := strings.Split(file, "/")
+		name := path[len(path)-2]
```

Comment:
Can we please check that `path` has at least a length of two? Otherwise this could panic and tear-down the complete telegraf instance.